packages/integration/src/azure/AzureIntegration.test.ts

```diff
@@ -41,4 +41,43 @@ describe('AzureIntegration', () => {
     expect(integration.type).toBe('azure');
     expect(integration.title).toBe('h.com');
   });
+
+  describe('resolveUrl', () => {
+    it('works for valid urls', () => {
+      const integration = new AzureIntegration({
+        host: 'dev.azure.com',
+      } as any);
+
+      expect(
+        integration.resolveUrl({
+          url: './a.yaml',
+          base:
+            'https://dev.azure.com/organization/project/_git/repository?path=%2Fcatalog-info.yaml',
+        }),
+      ).toBe(
+        'https://dev.azure.com/organization/project/_git/repository?path=%2Fa.yaml',
+      );
+
+      expect(
+        integration.resolveUrl({
+          url: 'https://absolute.com/path',
+          base:
+            'https://dev.azure.com/organization/project/_git/repository?path=%2Fcatalog-info.yaml',
+        }),
+      ).toBe('https://absolute.com/path');
+    });
+
+    it('falls back to regular URL resolution if there is no path param', () => {
+      const integration = new AzureIntegration({
+        host: 'dev.azure.com',
+      } as any);
+
+      expect(
+        integration.resolveUrl({
+          url: './a.yaml',
+          base: 'https://dev.azure.com/organization/project/_git/repository',
+        }),
+      ).toBe('https://dev.azure.com/organization/project/_git/a.yaml');
```

Comment:
Would expect this to be `https://dev.azure.com/organization/project/_git/repository?path=%2Fa.yaml` right?

Reply:
Good point - that's really a corner case I hadn't thought of. If you are at the root of the repo, doesn't it have a path of just `%2F`?

Reply:
It'll be a bit annoying to have to deduce and special treat the exact repo root - for all versions of azure - maybe git-url-parse can help somehoe

Reply:
Nah just not param. Maybe we can just throw if `path` is not present for now? It's similar to the GitHub example, except that that generally works a bit weird with folders too

Reply:
Updated! Did a middle-way thing that'll work for both cases.